.evergreen.yml

```diff
@@ -431,28 +443,16 @@ tasks:
       - func: compile_artifact
         vars:
           node_js_version: "14.15.1"
+
+  # PACKAGING
   - name: package_and_upload_artifact_macos
     depends_on:
       - name: check
         variant: darwin
-      - name: test_m40x_n12
-        variant: darwin
-      - name: test_m42x_n12
-        variant: darwin
-      - name: test_m44x_n12
-        variant: darwin
-      # Revert the commit that commented this out after resolving
+      # Remove the !.mlatest when this issue is resolved
       # https://jira.mongodb.org/browse/MONGOSH-592
-      #- name: test_mlatest_n12
-      #  variant: darwin
-      - name: test_m40x_n14
-        variant: darwin
-      - name: test_m42x_n14
-        variant: darwin
-      - name: test_m44x_n14
+      - name: ".unit-test !.mlatest"
```

Comment:
This means we're going to return to having failing runs in the waterfall, just without other tasks depending on them?

Reply:
Yes, this would have the red boxes back but they would not depend on it. So we'd just see when it works again. I could still leave them commented out of course!

Reply:
Ok -- I don't care much, but I think we'll see when this works again anyway when the SERVER ticket is resolved :)